container.go

```diff
@@ -351,6 +301,38 @@ func createContainer(pod *Pod, contConfig ContainerConfig) (*Container, error) {
 		c.mounts = mounts
 	}
 
+	return c, nil
+}
+
+func createContainers(pod *Pod, contConfigs []ContainerConfig) ([]*Container, error) {
```

Comment:
I wonder if this function should be renamed `createEmptyContainers()` as it is calling `newContainer()`, rather than `createContainer()`? Alternatively, maybe a comment on these functions would be useful to explain the differences.